platform/linux-generic/odp_pool.c

```diff
@@ -1556,7 +1582,9 @@ int odp_pool_stats(odp_pool_t pool_hdl, odp_pool_stats_t *stats)
 
 	pool = _odp_pool_entry(pool_hdl);
 
-	memset(stats, 0, sizeof(odp_pool_stats_t));
+	/* Don't overwrite input parameters */
+	memset(stats, 0, offsetof(odp_pool_stats_t, thread));
+	memset(stats->thread.cache_available, 0, sizeof(stats->thread.cache_available));
```

Comment:
It would be more robust to save first/last, memset(0) and then restore first/last. If new counter is added after thread struct, it is easy to forget to add another memset line here.

Reply:
Fixed in V2.